src/main/java/org/spongepowered/common/data/util/NbtDataUtil.java

```diff
@@ -196,6 +198,31 @@ public static void setLoreToNBT(ItemStack stack, List<Text> lore) {
         stack.getSubCompound(ITEM_DISPLAY, true).setTag(ITEM_LORE, list);
     }
 
+    public static boolean hasColorFromNBT(ItemStack stack) {
+        return stack.hasTagCompound() &&
+                stack.getTagCompound().hasKey(ITEM_DISPLAY) &&
+                stack.getTagCompound().getCompoundTag(ITEM_DISPLAY).hasKey(ITEM_COLOR);
+    }
+
+    public static Optional<Color> getColorFromNBT(NBTTagCompound subCompound) {
+        if (!subCompound.hasKey(ITEM_COLOR)) {
+            return Optional.empty();
+        }
+        return Optional.of(new Color(subCompound.getInteger(ITEM_COLOR)));
+    }
+
+    public static void removeColorFromNBT(ItemStack stack) {
+        if(stack.getSubCompound(ITEM_DISPLAY, false) == null) {
+            return;
+        }
+        stack.getSubCompound(ITEM_DISPLAY, false).removeTag(ITEM_COLOR);
+    }
+
+    public static void setColorToNbt(ItemStack stack, Color color) {
+        final int iColor = color.getRGB();
+        stack.getSubCompound(ITEM_DISPLAY, true).setInteger(ITEM_COLOR, iColor);
```

Comment:
Then a plugin/mod/update changes and then everybody wonders about strange behaviour. Consts stored in plugins does not work anymore. Or the value from get does not match the one you set a moment ago.